apps/coscrad-frontend/src/components/resources/spatial-features/test-utils/build-dummy-spatial-features.tsx

```diff
@@ -0,0 +1,38 @@
+import { GeometricFeatureType, ISpatialFeatureViewModel } from '@coscrad/api-interfaces';
+
+export const buildDummySpatialFeatures = (): ISpatialFeatureViewModel[] => [
```

Comment:
Eventually we can pull these from the same data we use to seed test \ demo data on the back-end. This will save us time over the long-haul but not so much immediately, so I suggest we wait.